scalableminds/webknossos
# Show prominent welcome banner on spotlight page

This PR adds a welcome banner for new users on the default page (spotlight). The banner encourages them to try out webknossos. The text of the banner has been copied from webknossos.org but is probably not what we want there and too scientific-y. Suggestions welcome (same for the background image and the general design).
The buttons link to the onboarding (get started), our public discuss instance (support) and our github repo (docs). The link to the docs should be replaced once we have real docs.

Currently the banner is always shown for logged out users. Ideally we would know in the frontend whether the `allowOrganzationCreation` flag in the backend is set and only show the banner in that case. @fm3 Maybe we could add that to the `/api/features` request?

![get-started-4](https://user-images.githubusercontent.com/1702075/43526272-16d4c508-95a4-11e8-9932-d4e18f7de517.png)


### URL of deployed dev instance (used for testing):
- https://showonboardingbanner.webknossos.xyz

### Steps to test:
- Visit the `/spotlight` page while being logged out. You should see the welcome banner. All buttons should work as expected.

### Issues:
- fixes #2933

------
